packages/core/content-manager/server/controllers/collection-types.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     const permissionQuery = await permissionChecker.sanitizedQuery.read(query);
 
     const populate = await getService('populate-builder')(model)
+      .populateFromQuery(permissionQuery)
```

Comment:
If there are permission with field condition(s) check AFTER query. Permission service will need these fields to do the job, or else it deny all output fields